components/FavoriteButton.js

```diff
@@ -0,0 +1,32 @@
+import { styled } from "styled-components";
+import { useState } from "react";
+
+export default function FavoriteButton({}) {
```

Comment:
```suggestion
export default function FavoriteButton({isFavourite, handleToggle}) {
```

Reply:
Du musst die Props hier rausreichen, wenn du sie in der PictureList weiterverarbeitest :)